src/fedora/39/amd64/Dockerfile

```diff
@@ -61,4 +61,10 @@ RUN dnf --setopt=install_weak_deps=False install -y \
 # Install ICU package to support globalization
 RUN dnf --setopt=install_weak_deps=False install -y \
         icu \
-    && dnf clean all
\ No newline at end of file
+    && dnf clean all
+
+# Dependencies for Aspnetcore
+ENV NODE_VERSION 20.9.0 
+RUN cd ~ && \ 
+    curl -LO https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz && \ 
+    tar xf node-v${NODE_VERSION}-linux-x64.tar.xz 
```

Comment:
The `tar` command is failing. I think it needs different options in Fedora than in Ubuntu.